fpmms/costing.py

```diff
@@ -0,0 +1,175 @@
+import tiktoken
+import anthropic
+from typing import Optional
+from mech.tools.prediction_request import prediction_request
+from mech.tools.prediction_request_sme import prediction_request_sme
+from mech.tools.prediction_request_claude import prediction_request_claude
+# from mech.tools.prediction_request_embedding import prediction_sentence_embedding
```

Comment:
Import will fail as these are not available in this project.

Reply:
the issue is there are dependency issues between the `scripts` repo and the `mech` repo so I am not able to use `mech` as a dependency.  Would you like me to include `mech` as a submodule?